packages/labs/ssr/src/lib/server-template.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {
+  html as baseHtml,
+  type CompiledTemplateResult,
+  type TemplateResult,
+} from 'lit-html';
+import {isServer} from 'lit-html/is-server.js';
+
+const SERVER_ONLY = 1;
+
+export interface ServerRenderedTemplate extends TemplateResult {
+  $_litServerRenderMode: typeof SERVER_ONLY;
```

Comment:
Should this brand follow the shareable private field pattern described here? https://github.com/lit/lit/blob/main/CONTRIBUTING.md#private-property-prefixing

And be similar to `_$litType$` ?
```suggestion
  _$litServerRenderMode$: typeof SERVER_ONLY;
```

Reply:
Oh, yes! Good catch!